openprescribing/api/views_measures.py

```diff
@@ -16,62 +17,52 @@ class MissingParameter(APIException):
 @api_view(['GET'])
 def measure_global(request, format=None):
     measure = request.query_params.get('measure', None)
-    tags = request.query_params.get('tags', None)
-    params = []
-    query = 'SELECT mg.month AS date, mg.numerator,  '
-    query += 'mg.denominator, mg.measure_id, '
-    query += 'mg.calc_value, mg.percentiles, mg.cost_savings, '
-    query += 'ms.name, ms.title, ms.description, '
-    query += 'ms.why_it_matters, ms.tags_focus, '
-    query += ' ms.denominator_short, ms.numerator_short, '
-    query += 'ms.url, ms.is_cost_based, ms.is_percentage, '
-    query += 'ms.low_is_good '
-    query += "FROM frontend_measureglobal mg "
-    query += "JOIN frontend_measure ms ON mg.measure_id=ms.id "
+    tags = request.query_params.getlist('tags', None)
```

Comment:
I don't think this does what you think it does.

If a request is to `/url/?q=1,2,3` then `request.GET.getlist('q')` will give `['1,2,3']`.  But it looks like you're expecting it to give `['1', '2', '3']`.

Reply:
aha, thanks